Makefile

```diff
@@ -452,14 +452,14 @@ update-dependencies: ## Uses go get -u to update all the dependencies while hold
 
 vet: ## Run mattermost go vet specific checks
 	@if ! [ -x "$$(command -v mattermost-govet)" ]; then \
-		echo "mattermost-govet is not installed. Please install it executing \"GO111MODULE=off go get -u github.com/mattermost/mattermost-govet\""; \
+		echo "mattermost-govet is not installed. Please install it executing \"GO111MODULE=off GOBIN=$(PWD)/bin go get -u github.com/mattermost/mattermost-govet\""; \
```

Comment:
@agnivade Does this require all developers that run 'make vet` to re-install `mattermost-govet`

Reply:
Yes indeed. That's a good point. The `command -v` check is incorrect. It should check for an executable at the GOBIN path. Let me change that.

Reply:
Would you please post a PSA after this PR has been merged in the developers channel?